mrmoody-metrics/config.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
if this is a template file, maybe it's better to save it as .dist or .template and the add the actual file to .gitignore ?